nrf52-hal-common/src/spim.rs

```diff
@@ -155,25 +156,58 @@ impl<T> Spim<T> where T: SpimExt {
             w.enable().enabled()
         );
 
-        // Set to SPI mode 0
-        spim.config.write(|w|
-            w
-                .order().msb_first()
-                .cpha().leading()
-                .cpol().active_high()
-        );
+        // Configure mode
+        spim.config.write(|w| {
+            // Can't match on `mode` as embedded_hal::spi::Mode only derives PartialEq, not Eq
```

Comment:
That's really annoying. Would you mind opening an issue upstream to add this derive?

Reply:
https://github.com/rust-embedded/embedded-hal/pull/126 My colleague appears to have already done so :)